packages/wrangler/src/__tests__/secret.test.ts

```diff
@@ -60,6 +60,22 @@ describe("wrangler secret", () => {
     describe("interactive", () => {
       useMockStdin({ isTTY: true });
 
+      test("should trim stdin secret value", async () => {
```

Comment:
```suggestion
      it("should trim stdin secret value", async () => {
```

Reply:
Also could we add a test for when `isTTY` is false?

Reply:
why did I use `test` 😅